packages/atlas/src/utils/data.ts

```diff
@@ -17,3 +17,17 @@ export const promisify =
   <T>(fn: (...args: unknown[]) => T) =>
   (...args: Parameters<typeof fn>) =>
     new Promise((resolve) => resolve(fn(...args))) as Promise<T>
+
+export const repeat = <T = undefined>(count: number, data?: T) => Array.from({ length: count }, () => data)
```

Comment:
I'm not too fond of the name of the function. I had to double-check what it does. Also, we're creating a video platform here, so repeat could be mistaken with some other function related to video playback. 
Maybe let's just name it `arrayFrom` or `arrayFromLength`? This is the first thing that comes to my mind.